.github/actions/retry/.config/jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
We generally fork and make changes when dealing with separate third-party repos, rather than incorporating them directly in oppia/oppia. Example: https://github.com/oppia/guppy

I've created https://github.com/oppia/retry for this purpose and made you a maintainer. Would you be able to use that instead, please?

Thanks!

Reply:
Ohhhh, that's so much easier!